package.json

```diff
@@ -42,7 +42,8 @@
         "build": "vite build",
         "storybook": "storybook dev -p 6006",
         "build-storybook": "storybook build",
-        "test": "jest"
+        "test": "jest",
+        "test:watch": "jest --watchAll"
```

Comment:
Let's also add this to https://github.com/powerhouse-inc/document-model-boilerplate.

This repo is supposed to be a fork of the boilerplate so we should try to keep these kind of changes in the boilerplate and then pull them to this one.